### PR TITLE
Updated Performance's empty state with new copy and image

### PR DESCRIPTION
**Pull request by silent1mezzo**
Previously:
Screen Shot 2020-06-26 at 4 37 07 PM

New:
empty-state-perf
empty-state-release

### CODE REVIEW

File: src/sentry/static/sentry/app/views/performance/onboarding.tsx [hunk 1]
```diff
@@ -1,28 +1,50 @@
...
+      </ButtonList>
+    </StyledOnboardingPanel>
   );
 }
 
+const StyledOnboardingPanel = styled(OnboardingPanel)`
+  background: url(${emptyState}) no-repeat left center !important;
+`;
+
+const ButtonList = styled('div')`
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/performance/onboarding.tsx:44]: You could use the `ButtonBar` component for this.

File: src/sentry/static/sentry/images/spot/performance-empty-state.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...43.87Zm13.23-.65a3.22,3.22,0,1,1-2-4.12A3.24,3.24,0,0,1,62.29,43.22Z" style="fill:#3f2b56"/><path d="M45.06,135l.48-.13s0,.05,0,.08Z" style="fill:#5c4376"/><path d="M40.25,40.76l-1.11.85-.77-1.13a.36.36,0,0,0-.52-.1.37.37,0,0,0-.1.52l.79,1.17-1.45,1.11a.37.37,0,0,0,.23.67.3.3,0,0,0,.22-.08L39,42.69l.93,1.37a.36.36,0,0,0,.31.16.39.39,0,0,0,.21-.06.37.37,0,0,0,.1-.52l-.95-1.41,1.15-.87a.39.39,0,0,0,.07-.53A.38.38,0,0,0,40.25,40.76Z" style="fill:#662429"/><path d="M51.1,40.59l-4-.19a.39.39,0,0,0-.39.36.37.37,0,0,0,.36.39l4,.19h0a.38.38,0,0,0,.38-.36A.38.38,0,0,0,51.1,40.59Z" style="fill:#231338"/><path d="M61.15,42.47l-1.33-.41.55-1.73a.37.37,0,1,0-.71-.22l-.55,1.73-1.87-.58A.37.37,0,1,0,57,42l1.86.58-.53,1.65a.38.38,0,0,0,.25.47l.11,0a.38.38,0,0,0,.36-.26l.53-1.66,1.33.41a.2.2,0,0,0,.11,0,.37.37,0,0,0,.11-.73Z" style="fill:#3f2b56"/><path d="M48.47,36.89a3.09,3.09,0,0,0-3.2,3,2.81,2.81,0,0,0,.92,2.07A2.89,2.89,0,0,1,46,40.82,3.09,3.09,0,0,1,51.52,39,3.16,3.16,0,0,0,48.47,36.89Z" style="fill:#231338"/><path d="M36.27,42.21a2.68,2.68,0,0,1,2.65-2.7,2.63,2.63,0,0,1,2.31,1.4,2.82,2.82,0,0,0-5.59.36,2.59,2.59,0,0,0,.8,1.85A2.49,2.49,0,0,1,36.27,42.21Z" style="fill:#471a20"/></g></g></svg>
```
Comment (k-fish) [inline src/sentry/static/sentry/images/spot/performance-empty-state.svg:1]: Has this asset been optimized? Looks like it still contains meta-data.

**Comment by doralchan** [comment]
@silent1mezzo This is great - we can get this out and fine tune the copy in another PR next week after Molly and Rahul add more to it.

File: src/sentry/static/sentry/app/views/performance/onboarding.tsx (hunk 1 above)
Comment (silent1mezzo) [reply src/sentry/static/sentry/app/views/performance/onboarding.tsx:43]: @markstory I'd need to style them anyways (`ButtonBar` button's go the full width). Is there any other value?

Comment (markstory) [reply src/sentry/static/sentry/app/views/performance/onboarding.tsx:52]: Using a styled button bar would be your best bet if you need a partial width container. Reusing the button bar would reduce some duplication and reduce the amount of generated CSS, but this works.

File: src/sentry/static/sentry/app/views/performance/onboarding.tsx [hunk 3]
```diff
@@ -1,28 +1,58 @@
...
 function Onboarding() {
   return (
-    <OnboardingPanel image={<IconLightning size="200px" />}>
-      <h3>{t('No transactions yet')}</h3>
+    <OnboardingPanel image={<PerfImage src={emptyStateImg} />}>
+      <h3>{t('Pinpoint problems')}</h3>
       <p>
         {t(
-          'View transactions sorted by slowest duration time, related issues, and number of users having a slow experience in one consolidated view. Trace those 10-second page loads to poor-performing API calls and its children.'
+          "You've got this souped up plan. Now what? Get your software set up. We’ve got transactions to track down."
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/performance/onboarding.tsx:17]: You have a mix of ascii quotes and apostrophes here, can we normalize on one of them?

**Review by markstory — APPROVED** [review]
Looks good

**Pull request merged**